numpy/lib/format.py

```diff
@@ -292,6 +292,10 @@ def _write_array_header(fp, d, version=None):
         None means use oldest that works
         explicit version will raise a ValueError if the format does not
         allow saving this data.  Default: None
+    fixedheaderlen: int, positive or zero, multiple of 16
+        If greater than zero, means the fixed number of bytes reserved for the array header in the binary file; useful if header
+        needs to be rewritten when some data is already present. Header will be padded with ' ' to specified length.
+        Must be a multiple of 16 to keep compatibility with padding convention. Default: 0
     Returns
```

Comment:
tuple-returns are a bit of an anti-pattern, but I guess it's fine for an internal function like this. However, we should update the docs to note the new return value